Rational() のテストで assertion にメッセージを渡して失敗した時になにが失敗した
のかわかるようにしています。
